dockyard/controllers/root.py

```diff
@@ -0,0 +1,29 @@
+from pecan import expose, rest
+
+from dockyard.controllers import v1
+
+class RootController(rest.RestController):
+
+    _version = ['v1']
+    # List of the allowed versions of the API.
+
+    _default_version = 'v1'
```

Comment:
We have to make decision whether we will allow docker client to work with our application. This implementations such that it can work, we have to decide whether we really want to support it. If it is so then we have to make sure that all the API versions are being supported. Therefor listing of docker API has to be done here to support it.